docs/unconstrained.rst

```diff
@@ -45,8 +46,10 @@ Instantiating and running the solver
 Continuing the ridge regression example above, gradient descent can be
 instantiated and run as follows::
 
-  gd = jaxopt.GradientDescent(fun=ridge_reg_objective, maxiter=500)
-  res = gd.run(init_params, l2reg=l2reg, X=X, y=y)
+  # solver = jaxopt.GradientDescent(fun=ridge_reg_objective, maxiter=500)
```

Comment:
is the commenting intentional? seems strange in the doc

Reply:
Yes it was to show how to instantiate other solvers. I added a comment to explain why.